nemo/collections/asr/models/configs/ctc_models_config.py

```diff
@@ -78,5 +78,5 @@ class EncDecCTCConfig(model_cfg.ModelConfig):
 
 
 @dataclass
-class EncDecCTCModelConfig(model_cfg.ModelPTConfig):
+class EncDecCTCModelConfig(model_cfg.ModelConfig):
```

Comment:
Since this is a model config, it doesnt need model: ..., all of the contents of `EncDecCTCConfig()` should be part of this since `EncDecCTCConfig()` is already a `ModelConfig`.

Two choices are - Rename this to `EncDecCTCModelPTConfig(model_cfg.NeMoModel)` or `EncDecCTCModelConfig(EncDecCTCConfig)`.

Reply:
changed to class EncDecCTCModelConfig(model_cfg.NemoConfig):